### ISSUE TITLE
fan regulation and questions

### ISSUE THREAD

**Issue by jmarcgit** [samuelolteanu/esphome-pellet-stove-controller#1]
Hi,

I find your idea to replace pellet stoves motherboards with an esp32 really smart. Especially because these  pellet boards are expensive and have limited features.

I have te replace mine soon. But my pellet stove is different. It's a cs thermos biomass model. There is no boiler, therefore there is an ambient fan (same triac as the exhaust fan) and a more advanced auge system.

A first auge just under the pellet tank and a second auge at the bottom, almost horizontal. The second auge runs all the time and has a feedback sensor. So  I just have to start it at boot and checks if it works time to time.

There is also a rotating pellet pot (it's a wheel actually) that must me moved time to time for cleanup. I need a relay for that also.

But I don' have water pump and related sensors on another hand.

I understand how the yaml works. Threre are some inputs thats use dac for reading temperature for example. And ouput pins to activate relays.

I have few questions:
-How to implement fan speed control ? Looks like you didn't use it right ?
-What is the regulation logic actually?

My plan:
I'm thinking to target an exhaust temperature (about 120 °C more or less) and modulate auge % and exhaust fan speed. I want the exhaust fan to be adjusted according (maybe) to the pressure sensor. And the auge % will give more or less heat so I can control the exhaust temperature with that.

In short I would like  to control the power just modulating the ambient fan speed. 

Thanks
